packages/functionals/botpress-qna/src/index.js

```diff
@@ -27,6 +28,12 @@ module.exports = {
       module: 'botpress-qna',
       type: 'incoming',
       handler: async (event, next) => {
+        if (typeof shouldProcessMessage === 'function') {
+          const state = await bp.dialogEngine.stateManager.getState(event.sessionId || event.user.id)
+          if (!await shouldProcessMessage(event, state)) {
```

Comment:
This will work for any falseish including `undefined`. I find it more conventional when such functions _must_ return `false` to stop propagation.

Reply:
@emirotin , OK, makes sense. Changed that